package.json

```diff
@@ -52,7 +52,6 @@
     "swagger-ui": "2.2.10",
     "toposort": "^1.0.0",
     "underscore": "~1.8.3",
-    "url-loader": "^0.5.7",
```

Comment:
We may want to leave this here, simply in case downstream plugins are expecting it to be present as a loader they can refer to.